custom/yeksi_naa_reports/filters.py

```diff
@@ -43,13 +43,14 @@ class MonthsDateFilter(BaseReportFilter):
 
     @classmethod
     def months(cls):
-        month_pairs = []
-        for month_number in range(1, 13):
-            month_pairs.append({
-                'name': calendar.month_name[month_number],
-                'value': month_number,
-            })
-        return month_pairs
+        return [
+            {u'name': 'Janvier', u'value': 1}, {u'name': 'Février', u'value': 2},
+            {u'name': 'Mars', u'value': 3}, {u'name': 'Avril', u'value': 4},
+            {u'name': 'Mai', u'value': 5}, {u'name': 'Juin', u'value': 6},
+            {u'name': 'Juillet', u'value': 7}, {u'name': 'Août', u'value': 8},
+            {u'name': 'Septembre', u'value': 9}, {u'name': 'Octobre', u'value': 10},
+            {u'name': 'Novembre', u'value': 11}, {u'name': 'Décembre', u'value': 12}
```

Comment:
could we just use [something like this](https://stackoverflow.com/a/13037425/8207) for this?

Reply:
I am not quite sure. This solution is dependent on allowed locales. I have found this type of solutions before, yet eg. for me they are throwing ```unsupported locale setting``` whenever trying to use the French language. From what I know it will throw this error for everything  that is not in ```locale -a```